charts/opentelemetry-collector/values.yaml

```diff
@@ -340,6 +346,7 @@ service:
   # loadBalancerIP: 1.2.3.4
   # loadBalancerSourceRanges: []
   annotations: {}
+  internalTrafficPolicy: ""
```

Comment:
We can set the default value here to `Cluster` since that is the default k8s value

Reply:
By the moment I commented it, because it is computed by the template for daemonsets

I did this, because If I set `Cluster` as default value, then people will set `mode: daemonset`, enable the creation of the service, and won't have to set explicitly `internalTrafficPolicy: Cluster` instead of `Local`

WDYT?

Reply:
Oh right. Let's allow this value in the schema then but not set it to anything, even empty string, by default. We can leave it commented out in the values.yaml.

Reply:
Hey, I already did it [in this change ](https://github.com/open-telemetry/opentelemetry-helm-charts/pull/755/files#diff-eba1f4eaa1181faf6ab17062aea0b2a92c389a268983766d94eb72f5ec8caa2fR350)

So I think it is ready to go :)